src/create-element.js

```diff
@@ -30,9 +30,13 @@ export function createElement(type, props, children) {
 			if (props[i] === undefined) props[i] = type.defaultProps[i];
 		}
 	}
-	let ref = props.ref;
+	let ref = null;
+	if (!(type != null && type._forwarded)) {
```

Comment:
We should not introduce the notion of forwarded in preact-core in my opinion, this is a principle that only exists in /compat

Reply:
I see, just wanted to remove the overhead of moving ref around unnecessarily.